specification/_global/msearch/types.ts

```diff
@@ -65,9 +65,5 @@ export class MultiSearchResult<TDocument> {
 
 /** @codegen_names result, failure */
 export type ResponseItem<TDocument> =
-  | MultiSearchItem<TDocument>
+  | SearchResponse<TDocument>
```

Comment:
multisearch response items have a `status` field that indicates the result of each search request. This is present in the [ES source code](https://github.com/elastic/elasticsearch/blob/8754c17a2d6218bce2041cddc572d2a58cef895f/server/src/main/java/org/elasticsearch/action/search/MultiSearchResponse.java#L164) and [the docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html#search-multi-search-api-response-body).